src/main/java/com/mongodb/hibernate/internal/translate/AbstractMqlTranslator.java

```diff
@@ -217,11 +223,35 @@ public void visitColumnWriteFragment(ColumnWriteFragment columnWriteFragment) {
     }
 
     // ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    // Table Mutation: deletion
+
+    @Override
+    public void visitStandardTableDelete(TableDeleteStandard tableDeleteStandard) {
+        if (tableDeleteStandard.getWhereFragment() != null) {
+            throw new FeatureNotSupportedException();
+        }
+
+        if (tableDeleteStandard.getNumberOfOptimisticLockBindings() > 0) {
+            throw new FeatureNotSupportedException("TODO-HIBERNATE-51 https://jira.mongodb.org/browse/HIBERNATE-51");
+        }
+
+        if (tableDeleteStandard.getNumberOfKeyBindings() > 1) {
+            throw new FeatureNotSupportedException("MongoDB doesn't support id spanning multiple columns");
+        }
+        assertTrue(tableDeleteStandard.getNumberOfKeyBindings() == 1);
+        var keyBinding = tableDeleteStandard.getKeyBindings().get(0);
+
+        var tableName = tableDeleteStandard.getMutatingTable().getTableName();
+        var astFilterField = new AstFilterField(keyBinding.getColumnReference().getColumnExpression());
+        var astValue = acceptAndYield(keyBinding.getValueExpression(), FIELD_VALUE);
```

Comment:
Here we read `getColumnExpression` at first, then `getValueExpression`. Let's have he same order in the `visitStandardTableInsert` method, to simplify comparing the logic. Currently, that method reads `getValueExpression` at first, then `getColumnExpression`.